src/index.js

```diff
@@ -0,0 +1,277 @@
+const tangramReference = require('tangram-reference').load();
+const color = require('./color.js');
+
+//getReferenceIndexedByCSS returns a reference based on ref indexed by the CSS property name instead of the reference original property name
+//it does a shallow copy of each property, therefore, it's possible to change the returned reference by changing the original one, don't do it
+function getReferenceIndexedByCSS(ref) {
+    var newRef = {};
+    for (var symb in ref.symbolizers) {
+        for (var property in ref.symbolizers[symb]) {
+            newRef[ref.symbolizers[symb][property].css] = ref.symbolizers[symb][property];
+        }
+    }
+    return newRef;
+}
+const referenceCSS = getReferenceIndexedByCSS(tangramReference);
+
+
+function wrapFn(functionBody) {
+    return 'function (){var data=feature; var ctx={zoom: $zoom};' + functionBody + '}';
+}
+function getReferenceDefault(property) {
+    return referenceCSS[property]['default-value'];
+}
+
+function getLiteralFromShaderValue(shaderValue) {
+    // jshint ignore:start
+    // required within the eval context
+    const ctx = { zoom: 10 };
+    // jshint ignore:end
+    var _value = null;
+    shaderValue.js.forEach(function (code) {
+        eval(code);
+    });
+    return _value;
+}
+
+//Returns the final opacity override selecting between fill-opacity and outline-opacity.
+//Returned value can be a float or a function string to be called at Tangram's runtime if the override is active
+//A falseable value will be returned if the override is not active
+function getOpacityOverride(yamlDrawGroup, isFill) {
+    var opacity;
+    if (isFill) {
+        opacity = yamlDrawGroup._hidden['opacity:fill'];
+    } else {
+        opacity = yamlDrawGroup._hidden['opacity:outline'];
+    }
+    opacity = yamlDrawGroup._hidden['opacity:general'] || opacity;
+    return opacity;
+}
+
+function isNumeric(n) {
+    return !isNaN(parseFloat(n)) && Number.isFinite(n);
```

Comment:
This is exactly the same as doing `Number.isFinite(n)` alone, isn't it?

Reply:
Good point! Yes, is redundant, and there is another problem, you mentioned before the polyfill for `startsWith`, but this could be a problematic function too. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite it isn't supported on IE